WcaOnRails/env_config.rb

```diff
@@ -22,6 +22,7 @@
     mandatory :VAULT_APPLICATION, :string
     mandatory :VAULT_AWS_REGION, :string
     mandatory :INSTANCE_ROLE, :string
+    mandatory :WCA_REGISTRATION_URL, :string
```

Comment:
Where is the value for this variable being populated? --> Chef
How is it being defined in non-production environments? --> Second optional declaration below (and possible changes to our development env files)

Reply:
Have we talked about how to handle wca-registrations being an external microservice in local development?

Reply:
Oh heavens no :scream: